ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -370,33 +384,47 @@ const GENE_DETAIL_SECTIONS = [
     color: 'pink',
     description: 'RNA-Seq Outlier',
     label: 'RNA-Seq',
-    showDetails: (gene, rnaSeqData) => rnaSeqData && rnaSeqData[gene.geneId],
-    detailsDisplay: (gene, rnaSeqData) => (
-      <div>
-        This gene is flagged as an outlier for RNA-Seq in the following samples
-        <Table basic="very" compact="very">
-          <Table.Header>
-            <Table.Row>
-              <Table.HeaderCell />
-              {RNA_SEQ_DETAIL_FIELDS.map(
-                field => <Table.HeaderCell key={field}>{camelcaseToTitlecase(field).replace(' ', '-')}</Table.HeaderCell>,
-              )}
-            </Table.Row>
-          </Table.Header>
-          <Table.Body>
-            {Object.entries(rnaSeqData[gene.geneId]).map(([individual, data]) => (
-              <Table.Row key={individual}>
-                <Table.HeaderCell>{individual}</Table.HeaderCell>
-                {RNA_SEQ_DETAIL_FIELDS.map(
-                  field => <Table.Cell key={field}>{data[field].toPrecision(3)}</Table.Cell>,
-                )}
-              </Table.Row>
-            ))}
-          </Table.Body>
-        </Table>
-      </div>
+    showDetails: (gene, { rnaSeqData }) => rnaSeqData && rnaSeqData[gene.geneId],
+    detailsDisplay: (gene, { rnaSeqData }) => (
+      <DataTable
+        basic="very"
```

Comment:
why did you remove `compact="very"`

Reply:
Do you mean the prop for the `DataTable`? I can't find the `compact` in the prop list of DataTable.

Reply:
it passes through props to the table so you can pass any table prop to it. Since you have to maintain the existing functionality, you need to pass the same props